shakedown/dcos/package.py

```diff
@@ -5,7 +5,7 @@
 from dcoscli.package.main import _get_cosmos_url
 
 import shakedown
-
+from shakedown import *
```

Comment:
I'm not a python dev (yet).  I needed functions in __init__ and this seems to be the way to get that to work.  I'm not sure if the import on line 7 is necessary.